lib/server/idemix/nonce.go

```diff
@@ -145,7 +145,6 @@ func (nm *nonceManager) startNonceSweeper() {
 func (nm *nonceManager) sweep(curTime time.Time) error {
```

Comment:
This entire function `sweep` seems useless. The two calls to it should just be replaced with a call to `removeExpiredNoncesFromDB` directly